### PR TITLE
WIP PR for BXMSDOC-6281: Update the managing & monitoring KIE Server document as per RHPAM/RHDM 7.9 release.

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRA:**
- [Epic](https://issues.redhat.com/browse/BXMSDOC-6279l)
- [Dedicated JIRA](https://issues.redhat.com/browse/BXMSDOC-6281)

**Doc previews:**

- [I. Managing and monitoring KIE Server-RHPAM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHPAM-MMKS-Latest/#kie-server-configure-central-proc_execution-server)
- [I. Managing and monitoring KIE Server-RHDM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHDM-MMKS-Latest/#kie-server-configure-central-proc_execution-server)

**Doc impact:** 

- Please check **chapter 9. Configuring a KIE Server to connect to Business Central** from **RHPAM** doc preview.
- Please check **chapter 8. Configuring a KIE Server to connect to Business Central** from **RHDM** doc preview.

As per your comment in the BXMSDOC-6281 JIRA, these three chapters are similar so I have clubbed the following chapters:

- Configuring KIE Server with the integrated Decision Manager controller
- Configuring a KIE Server to connect to Business Central 
- Configuring KIE Server Managed by Business Central

And consolidated as one single chapter: **Configuring a KIE Server to connect to Business Central**. 

The same section is present in the following documents: 

1. Please check **chapter 14.5. Configuring KIE Server to connect to Business Central**.

- [Installing and configuring Red Hat Process Automation Manager on Red Hat JBoss EAP 7.3](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHPAM-EAP-Latest/#kie-server-configure-central-proc_install-on-eap)
- [Installing and configuring Red Hat Decision Manager on Red Hat JBoss EAP 7.3](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHDM-EAP-Latest/#kie-server-configure-central-proc_install-on-eap)

2. Please check **chapter 2.1. Configuring KIE Server to connect to Business Central**.

- [Packaging and deploying a Red Hat Process Automation Manager project](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHPAM-PD-Latest/#kie-server-configure-central-proc_packaging-deploying)
- [Packaging and deploying a Red Hat Decision Manager project](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6281-RHDM-PD-Latest/#kie-server-configure-central-proc_packaging-deploying)

### CODE REVIEW

**Comment by kaldesai** [comment]
Hey @hmanwani-rh, Can you please review and verify the doc content? I have attached the latest doc preview links in the description.

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,170 @@
+[id='kie-server-configure-central-proc_{context}']
+= Configuring {KIE_SERVER} to connect to {CENTRAL}
+
+WARNING: This section provides a sample setup that you can use for testing purposes. Some of the values are unsuitable for a production environment, and are marked as such.
+
+If a {KIE_SERVER} is not already configured in your {PRODUCT} environment, or if you require additional {KIE_SERVERS} in your {PRODUCT} environment, you must configure a {KIE_SERVER} to connect to {CENTRAL}.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:6]:
```suggestion
If a {KIE_SERVER} is not configured in your {PRODUCT} environment, or if you require additional {KIE_SERVERS} in your {PRODUCT} environment, you must configure a {KIE_SERVER} to connect to {CENTRAL}.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+[NOTE]
+====
+If you are deploying {KIE_SERVER} on {OPENSHIFT}, see the {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_] document for instructions about configuring it to connect to {CENTRAL}.
+====
+
+ifeval::["{context}" == "execution-server"]
+{KIE_SERVER} can be managed or it can be unmanaged. If {KIE_SERVER} is unmanaged, you must manually create and maintain KIE containers (deployment units). If {KIE_SERVER} is managed, the {CONTROLLER} manages the {KIE_SERVER} configuration and you interact with the {CONTROLLER} to create and maintain KIE containers.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:14]:
```suggestion
{KIE_SERVER} can be managed or unmanaged. If {KIE_SERVER} is unmanaged, you must manually create and maintain KIE containers (deployment units). If {KIE_SERVER} is managed, the {CONTROLLER} manages the {KIE_SERVER} configuration and you interact with the {CONTROLLER} to create and maintain the KIE containers.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+[NOTE]
+====
+Only make the changes described in this section if {KIE_SERVER} will be managed by {CENTRAL} and you installed {PRODUCT} from the ZIP files.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:18]: This section contains various admonitions, I'd suggest changing this as normal text. 

```suggestion
Make the changes described in this section if {KIE_SERVER} is managed by {CENTRAL} and you have installed {PRODUCT} from the ZIP files.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+If you did not install {CENTRAL}, you can use the {HEADLESS_CONTROLLER} to manage {KIE_SERVER}, as described in xref:controller-con_{context}[].
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:19]:
```suggestion
If you have installed {CENTRAL}, you can use the {HEADLESS_CONTROLLER} to manage {KIE_SERVER}, as described in xref:controller-con_{context}[].
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
++
+For more information, see <<eap-users-create-proc_install-on-eap>>.
+endif::[]
+
+ifeval::["{context}" == "execution-server"]
+* {CENTRAL} and {KIE_SERVER} are installed in the base directory of the {EAP} installation (`__EAP_HOME__`).
+
+[NOTE]
+====
+You should install {CENTRAL} and {KIE_SERVER} on different servers in production environments. In this sample situation, we use only one user named `controllerUser` that has both the `rest-all` and the `kie-server` roles. However, if you install {KIE_SERVER} and {CENTRAL} on the same server, for example in a development environment, make the changes described in this section in the shared `standalone-full.xml` file.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:42]:
```suggestion
You must install {CENTRAL} and {KIE_SERVER} on different servers in production environments. In this sample situation, we use only one user named `controllerUser`, containing both `rest-all` and the `kie-server` roles. However, if you install {KIE_SERVER} and {CENTRAL} on the same server, for example in a development environment, make the changes in the shared `standalone-full.xml` file as described in this section.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+** On the {KIE_SERVER}, a user with the role `kie-server`
+
+endif::[]
+
+ifeval::["{context}" == "packaging-deploying"]
+* {CENTRAL} and {KIE_SERVER} are installed in the base directory of the {EAP} installation (`__EAP_HOME__`).
+
+[NOTE]
+====
+You should install {CENTRAL} and {KIE_SERVER} on different servers in production environments. In this sample situation, we use only one user named `controllerUser` that has both the `rest-all` and the `kie-server` roles. However, if you install {KIE_SERVER} and {CENTRAL} on the same server, for example in a development environment, make the changes described in this section in the shared `standalone-full.xml` file.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:56]:
```suggestion
You must install {CENTRAL} and {KIE_SERVER} on different servers in production environments. In this sample situation, we use only one user named `controllerUser`, containing both the `rest-all` and the `kie-server` roles. However, if you install {KIE_SERVER} and {CENTRAL} on the same server, for example in a development environment, make the changes in the shared `standalone-full.xml` file as described in this section.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+
+[source,xml,subs="attributes+"]
+----
+<property name="org.kie.server.user" value="controllerUser"/>
+<property name="org.kie.server.pwd" value="controllerUser1234;"/>
+----
+
+--
++
+. Verify the successful start of the {KIE_SERVER} by sending a GET request to `http://_SERVER:PORT_/kie-server/services/rest/server/`.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:138]:
```suggestion
. Send a GET request to `http://_SERVER:PORT_/kie-server/services/rest/server/` and verify the successful start of the {KIE_SERVER}.
```

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+Once authenticated, you get an XML response similar to this:
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:139]:
```suggestion
Once authenticated, you receive an XML response similar to the following example:
```

**Review by hmanwani-rh — CHANGES_REQUESTED** [review]
@kaldesai Please see the comments below and let me know if you have any questions.

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:6]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:14]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:18]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 4 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:19]: Done.

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 5 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:42]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 6 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:56]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 8 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:139]: Done

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 7 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:138]: Done.

**Pull request merged**